mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -58,6 +66,7 @@
   $: columnId = column.id;
   $: ({ activeCell, selectedCells } = selection);
   $: isActive = $activeCell && isCellActive($activeCell, row, processedColumn);
+  $: ({ 1: rId } = row.record);
```

Comment:
As mentioned on the other PR, the first item is not the primary key. It is not always the `id` column, and there is no guarantee that an id column will be present, since Mathesar also supports DBs and tables created outside of Mathesar.

Refer the `Row` component where there's a `primaryKeyColumnId` property. Similarly refer the `RowControl` component on how to obtain the value of the primary key column from the row.

Reply:
Done.